app/views/host_aggregate/edit.html.haml

```diff
@@ -1,36 +1,3 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}
```

Comment:
looks like this old code is coming from an angular controller, so please find that file and remove if it is unnecessary.

Reply:
add_host_select.html.haml and remove_host_select.html.haml also both reference `%form#form_div{:name => "angularForm", 'ng-controller' => "hostAggregateFormController"}`. Should i delete host_aggregate_form_controller.js as well as these other two files? From what i can tell they are not being used by the code anymore.

Reply:
@MelsHyrule what are the other two files?

Reply:
app/views/host_aggregate/add_host_select.html.haml and app/views/host_aggregate/remove_host_select.html.haml

Reply:
i can see this code
```
  def remove_host_select
    assert_privileges("host_aggregate_remove_host")
    @host_aggregate = find_record_with_rbac(HostAggregate, params[:id])
    @in_a_form = true
    @host_choices = {}
    @host_aggregate.hosts.each do |host|
      @host_choices["#{host.name}: #{host.hostname}"] = host.id
    end

    if @host_choices.empty?
      add_flash(_("No hosts to remove from Host Aggregate \"%{name}\"") % {
        :name => @host_aggregate.name
      }, :error)
      session[:flash_msgs] = @flash_array
      @in_a_form = false
      redirect_to(last_screen_url)
    else
      drop_breadcrumb(
        :name => _("Remove Host from Host Aggregate \"%{name}\"") % {:name => @host_aggregate.name},
        :url  => "/host_aggregate/remove_host/#{@host_aggregate.id}"
      )
    end
  end

Reply:
so, keep those files and delete only unnecessary code from controller.

Reply:
in host aggregates from, you can see the options in configuration dropdown.
Add Host to selected....
Remove Host to selected...

Reply:
remove `def remove_host_select` and `def add_host_select` blocks from host_aggregate_controller.rb then?

Reply:
why do you remove those? you need to keep those for those two forms to work.

Reply:
So it would be just remove the js file?

Reply:
Do not remove the file, it is being used two other places, only remove unused code in the controller.